docs/components/docgen.js

```diff
@@ -25,10 +25,16 @@ export type DocGen = {|
   |},
 |};
 
-export default async function docgen(component: string): Promise<DocGen> {
+export default async function docgen({
+  componentName,
+  alternativeSubdirectory = '',
+}: {|
+  componentName: string,
+  alternativeSubdirectory?: string,
+|}): Promise<DocGen> {
   const filePath = path.join(
     nextConfig().serverRuntimeConfig.GESTALT_ROOT,
-    `/packages/gestalt/src/${component}.js`,
+    alternativeSubdirectory || `/packages/gestalt/src/${componentName}.js`,
```

Comment:
@AlbertCarreras this is a code smell - could we either

* Avoid this extra parameter by moving `OnLinkNavigationProvider` or `ColorSchemeProvider`?
* Make the import explicit for every component?